irods_testing_environment/test_manager.py

```diff
@@ -11,6 +11,14 @@ class test_manager:
     def __init__(self, containers, tests, test_type='irods_python_suite'):
         """Constructor for `test_manager`.
 
+        A note about passing None to `tests`:
+        None is a special value which indicates that no test list was specified. The test
```

Comment:
I noticed that all references to **None** are surrounded by backticks except the one on this line.

Reply:
Done

Reply:
I actually was referring to L15.

"None is a special ..."

Reply:
Whoops. Now done.